core/src/main/java/org/geysermc/geyser/text/MinecraftLocale.java

```diff
@@ -141,25 +148,46 @@ private static Path getPath(String locale) {
      * @param locale Locale to load
      */
     private static boolean loadLocale(String locale) {
-        File localeFile = GeyserImpl.getInstance().getBootstrap().getConfigFolder().resolve("locales/" + locale + ".json").toFile();
+        String bedrockLocale = locale.toLowerCase(Locale.ROOT);
+        if (bedrockLocale.equals("no_no")) {
+            // Store this locale under the Bedrock locale so we don't need to do this check over and over
+            bedrockLocale = "nb_no";
+        }
+
+        Map<String, String> langMap = new HashMap<>();
 
         // Load the locale
-        if (localeFile.exists()) {
-            // Read the localefile
-            InputStream localeStream;
-            try {
-                localeStream = new FileInputStream(localeFile);
-            } catch (FileNotFoundException e) {
-                throw new AssertionError(GeyserLocale.getLocaleStringLog("geyser.locale.fail.file", locale, e.getMessage()));
-            }
+        Path localeFile = getPath(bedrockLocale);
```

Comment:
This too will fail for nb_no, as the downloaded locale is saved under the java name `no_no` - so this needs to be moved to before we replace the locale. We still need to replace it since the `LOCALE_MAPPINGS` map takes in the bedrock locale as the keys, but the downloaded java lang files are saved under the java name.